www/js/services.js

```diff
@@ -19,21 +19,55 @@ angular.module('smartAlarm.services', [])
   };
 })
 
+.factory('Notification', function($cordovaLocalNotification){
+
+  var setAlarmTime = function(data) {
+    var hours = parseInt(data.time_to_leave.substr(0,2));
+    var minutes = parseInt(data.time_to_leave.substr(3,4));
+    return new Date().setHours(hours, minutes);
+  };
+
+  var add = function(data) {
+    var alarmTime = setAlarmTime(data);
+    $cordovaLocalNotification.add({
+        id: "1234",
+        date: alarmTime,
+        message: "It's time to go!",
+        title: "LEAVE",
+        autoCancel: true,
+        sound: 'file://assets/Drop-what-youre-doing-and-leave-now.mp3'
+    }).then(function () {
+        alert("Your alarm to LEAVE has been set!");
+    });
+  };
+  return add;
+})
+
 .service('StationList', function($http) {
-  return $http.get('/stations');
+  return $http.get('https://makers-alarm.herokuapp.com/stations');
 })
 
 .service('CurrentWeather', function($http) {
-  return $http.get('/api/weather');
+  return $http.get('https://makers-alarm.herokuapp.com/weather_api');
 })
 
-.service('GetTrip', function($http) {
+.service('PostTrip', function($http) {
   return function (tripDetails) {
       return $http({
         method: 'POST',
-        url: '/alarms',
+        url: 'https://makers-alarm.herokuapp.com/alarms',
         contentType: 'application/json',
         data: tripDetails
       });
   };
+})
+
+.service('GetTrip', function($http) {
+  return function (tripDetails) {
+      return $http({
+        method: 'GET',
+        url: 'https://makers-alarm.herokuapp.com/alarms',
+        contentType: 'application/json'
+      });
+  };
```

Comment:
we should decide wether we want to use $http or $resource since our services are now using both and they're variants of the same thing